src/recordings.js

```diff
@@ -11,3 +11,103 @@ export async function fetchFirstRecordingURL(wordform) {
   let results = await fetchRecordings(wordform)
   return results[0]['recording_url']
 }
+
+
+/**
+ * Render a list of speakers for the user. The speakers' names can be clicked and the user can hear the word in a variety of dialects (if that's the proper word!).
+ *
+ * @param wordform {string} – the term being defined
+ * 
+ */
+export function getSpeakerList(wordform) {  
+  // get the value of the wordform from the page
+  wordform = document.getElementById('data:head').value;
+  const BASE_URL = 'http://sapir.artsrn.ualberta.ca/validation/recording/_search/' + `${wordform}`;
+
+  // setting up the JSON request
+  let xhttp = new XMLHttpRequest();
+  xhttp.open('GET', BASE_URL, true);
+
+  // receiving request information from SAPIR
+  xhttp.onload = function() {
+    let returnedData = JSON.parse(this.response);
+    let numberOfSpeakers = returnedData.length;
+    const recordingsList = document.querySelector('.recordings');
+    const recordingsHeading = document.querySelector('.definition__recordings');
+    
+    // we only want to display our list of speakers once!
+    if (recordingsList.childElementCount < numberOfSpeakers) {
+      displaySpeakerList(returnedData);
+    }
+
+    // the function that displays an individual speaker's name
+    function displaySpeakerList(firstJSONData) {
+      recordingsHeading.insertAdjacentHTML('afterbegin', '<h3 class="explainer">Tap the names below to hear the word said by various speakers.</h3>');
+      let speakerListIndex = 0;
+      
+      
+      while (speakerListIndex < numberOfSpeakers) {
+        const individualSpeaker = document.createElement('li');
+    
+        // the value of each list item is actually a loop through the names of the speakers for the particular wordform
+        individualSpeaker.innerHTML = '<a class="audio-snippet">' + firstJSONData[speakerListIndex].speaker_name + '</a>';
```

Comment:
Given that the interaction with this component initiates an action, and does NOT initiate navigation, this should be a `<button>`.  For a fun video on why, I recommend this video! https://www.youtube.com/watch?v=8XjwDq9zG4I